xchange-bitmex/src/main/java/org/knowm/xchange/bitmex/BitmexExchange.java

```diff
@@ -27,8 +27,8 @@
 
 public class BitmexExchange extends BaseExchange implements Exchange {
 
-  private final SynchronizedValueFactory<Long> nonceFactory = new ExpirationTimeFactory(30);
   protected RateLimitUpdateListener rateLimitUpdateListener;
+  private SynchronizedValueFactory<Long> nonceFactory = new ExpirationTimeFactory(30);
```

Comment:
why is this not final any more?

Reply:
It is for me, or at least in latest version

Reply:
found it on another change list